modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/BaseDataDefinitionResourceTestCase.java

```diff
@@ -722,7 +722,6 @@ public void testPostDataDefinitionCopy() throws Exception {
 		DataDefinition postDataDefinition =
 			testPostDataDefinitionCopy_addDataDefinition(randomDataDefinition);
 
-		assertEquals(randomDataDefinition, postDataDefinition);
 		assertValid(postDataDefinition);
```

Comment:
@devpedrotavares We should not change the `BaseDataDefinitionResourceTestCase` class, because is autogenerated..

Reply:
Fixed it!